StripeCryptoOnramp/StripeCryptoOnramp/Source/Components/CryptoOnrampCoordinator.swift

```diff
@@ -24,6 +25,23 @@ public protocol CryptoOnrampCoordinatorProtocol {
     /// - Parameter email: The email address of the user in question.
     /// - Returns: Whether the email corresponds to an already registered Link user.
     func isLinkUser(email: String) async throws -> Bool
+
+    /// Registers a new Link user with the provided details.
+    /// `isLinkUser` must be called before this.
+    ///
+    /// - Parameter fullName: The full name of the user.
+    /// - Parameter phone: The phone number of the user. Expected to be in E.164 format.
+    /// - Parameter country: The country code of the user.
+    /// Throws if `isLinkUser` was not called prior to this, or an API error occurs.
+    func registerNewLinkUser(phone: String, country: String, fullName: String?) async throws
```

Comment:
There's no success or failure return value, but it will throw in the case of failure. If there's still a desire to return something in the non-throwing case, I could return an always `true` `Bool`. Just let me know if that'd be preferred.